notifications/src/main/java/org/openedx/notifications/presentation/inbox/NotificationsInboxFragment.kt

```diff
@@ -0,0 +1,483 @@
+        )
+
+        IconButton(
+            modifier = Modifier.align(Alignment.CenterEnd),
+            onClick = { onSettingsClick() }
+        ) {
+            Icon(
+                imageVector = Icons.Default.MoreVert,
+                tint = MaterialTheme.appColors.primary,
+                contentDescription = stringResource(id = R.string.notifications_accessibility_settings)
+            )
+        }
+    }
+}
+
+@Composable
+private fun SectionHeader(
+    modifier: Modifier = Modifier,
+    section: InboxSection,
+) {
+    Row(
+        modifier = modifier,
+        horizontalArrangement = Arrangement.Start,
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        Text(
+            text = stringResource(id = section.titleResId),
+            modifier = Modifier.padding(bottom = 24.dp),
+            style = MaterialTheme.appTypography.titleSmall,
+            color = MaterialTheme.appColors.textPrimaryVariant,
+        )
+    }
+}
+
+@Composable
+private fun NotificationItemView(
+    modifier: Modifier = Modifier,
+    item: NotificationItem,
+) {
+    Row(
+        modifier = modifier.padding(vertical = 8.dp),
+        horizontalArrangement = Arrangement.spacedBy(8.dp),
+        verticalAlignment = Alignment.Top,
+    ) {
+        Icon(
+            imageVector = Icons.Outlined.Forum,
+            contentDescription = null,
+            tint = MaterialTheme.appColors.textPrimary,
+        )
+        Column(
+            modifier = Modifier.weight(1f),
+            verticalArrangement = Arrangement.spacedBy(8.dp)
+        ) {
+            Row(
+                horizontalArrangement = Arrangement.spacedBy(8.dp)
+            ) {
+                Text(
+                    modifier = Modifier.weight(1f),
+                    text = item.content,
+                    style = MaterialTheme.appTypography.bodyMedium,
+                    color = MaterialTheme.appColors.textPrimary,
+                )
+                if (item.lastRead.isNull()) {
+                    Box(
+                        modifier = Modifier
+                            .clip(CircleShape)
+                            .align(Alignment.CenterVertically)
+                            .size(8.dp)
+                            .background(MaterialTheme.appColors.primaryButtonBackground)
+                    )
+                }
+            }
+            Text(
+                text = TextUtils.dateToRelativeTimeString(LocalContext.current, item.created),
+                style = MaterialTheme.appTypography.bodySmall,
+                color = MaterialTheme.appColors.notificationTimestamp,
+            )
+        }
+    }
+}
+
+@Composable
+fun InboxStateView(
+    modifier: Modifier = Modifier,
+    uiState: InboxUIState,
+    onReloadNotifications: () -> Unit = { },
+) {
+    val iconResId = if (uiState is InboxUIState.Empty) Icons.Outlined.Notifications
+    else Icons.Outlined.SignalWifiStatusbarConnectedNoInternet4
```

Comment:
The icon color didn't match with the [Figma designs](https://www.figma.com/design/I9xgrJzdTje5vKgh6DD57C/Mobile-UI-Screens-(2025)?node-id=116-3032&t=EnpEWGB5oFiTm5hh-0)

Reply:
I know that. I intentionally used the color already used in the app so we wouldn't have to introduce new colors.